### PR TITLE
Fix natspec comments

### LINKED ISSUE #8: Have 100% unit test coverage

### PR DESCRIPTION
**Pull request by dan13ram**
Closes #8

### CODE REVIEW

File: contracts/QuestChain.sol [hunk 1]
```diff
@@ -69,89 +109,118 @@ contract QuestChain is
...
 
+    /**
+     * @dev Creates quests in quest chain
+     * @param _detailsList list of uris of off chain details for new quests
+     */
     function createQuests(string[] calldata _detailsList)
         external
         onlyRole(EDITOR_ROLE)
     {
-        uint256 _loopLength = _detailsList.length;
```
Comment (parv3213) [inline contracts/QuestChain.sol:128]: @dan13ram why did we remove this?

File: package.json [hunk 2]
```diff
@@ -60,6 +60,7 @@
     "solhint": "^3.3.7",
     "solhint-plugin-prettier": "^0.0.5",
     "solidity-coverage": "^0.7.21",
+    "solmate": "^6.5.0",
```
Comment (parv3213) [inline package.json]: where are do we use this?

**Review by parv3213 — APPROVED** [review]

File: contracts/QuestChain.sol (hunk 1 above)
Comment (dan13ram) [reply contracts/QuestChain.sol:128]: Its unnecessary. simply extra gas waste

File: package.json (hunk 2 above)
Comment (dan13ram) [reply package.json]: In the mock erc20 contract.

Comment (dan13ram) [reply package.json]: Removed the dependency in [0479366](https://github.com/quest-chains/contracts/pull/13/commits/047936694a1112467b7bc10f266c891dedc2f79f)

**Pull request merged**